media-libs/avbin-bin/avbin-bin-8.1.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="A thin wrapper around FFmpeg"
+HOMEPAGE="http://avbin.github.io/"
+SRC_URI="
+	amd64? ( mirror://github/AVbin/AVbin/avbin-linux-x86-64-v${PV}.tar.bz2 )
+	x86? ( mirror://github/AVbin/AVbin/avbin-linux-x86-32-v${PV}.tar.bz2 )"
+
+LICENSE="GPL-3 LGPL-3"
```

Comment:
The source build [is a mess](663#issuecomment-243397438).

Is there any way we can "make the source accessible"? AVbin isn't entirely deprecated yet, and `sci-biology/psychopy` video playback (which depends on it) [works considerably better with `=avbin-8.1`](http://discourse.psychopy.org/t/sound-quality-for-moviestim-is-very-bad/955/4?u=horea_christian) - so I think we should definitely bump this independently of whether or not we end up providing a source package.